travel_platform/skyscanner_data/skyscanner_data.py

```diff
@@ -15,57 +15,41 @@
 inbound_date = config.get('api_skyscanner', 'inbound_date')
 flights_cache_service = FlightsCache(api_key)
 
+#FIXME: why my *args not work?
```

Comment:
Well, `FIXME` should describe a problem.
If there are some questions, it may be done as an additional part of this `FIXME`.